core/services/aws-ses/src/main/java/com/mongodb/stitch/core/services/aws/ses/CoreAwsSesServiceClient.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2018-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.stitch.core.services.aws.ses;
+
+import com.mongodb.stitch.core.services.internal.CoreStitchService;
+import java.util.Collections;
+import org.bson.Document;
+
+public class CoreAwsSesServiceClient {
+
+  private final CoreStitchService service;
+
+  protected CoreAwsSesServiceClient(final CoreStitchService service) {
+    this.service = service;
+  }
+
+  protected void sendEmailInternal(
+      final String toAddress,
+      final String fromAddress,
+      final String subject,
+      final String body
+  ) {
+    final Document args = new Document();
```

Comment:
[minor] if I'm being a stickler, we can pull these keys out into consts, or, create a codec for this structure.

Reply:
I think they don't really belong anywhere but here if they're used just once.

Reply:
Alright, disregard then